extraction/src/test/scala/org/broadinstitute/monster/encode/extraction/MockEncodeClient.scala

```diff
@@ -1,25 +1,20 @@
 package org.broadinstitute.monster.encode.extraction
 
-import upack.Msg
+import upack._
 
 import scala.concurrent.Future
 import scala.collection.mutable
 
 /** Mock interface for ENCODE API. */
 class MockEncodeClient(
-  responseMap: Map[(EncodeEntity, (String, String)), Msg]
+  responseMap: Map[(EncodeEntity, (String, String)), List[Msg]]
 ) extends EncodeClient {
   val recordedRequests = mutable.Set[(EncodeEntity, (String, String))]()
 
   override def get(entity: EncodeEntity, params: List[(String, String)]): Future[Msg] = {
-    val paramsToUse = params.head
```

Comment:
This `.head` was only safe when we were using batch size = 1. I see we bumped to a size of 2 at some point, which is probably good to test anyways, so I reworked this to deal with any number of input params.